user-manual/metadata/premis.rst

```diff
@@ -4,11 +4,232 @@
 PREMIS metadata in Archivematica
 ================================
 
-Archivematica supports the entry of `PREMIS <http://www.loc.gov/standards/premis/>`_
-rights metadata during :ref:`transfer <transfer>` or :ref:`ingest <ingest>`.
-Rights entered via the GUI interface apply to the entirety of the SIP or transfer.
-Rights can also be entered at the object level by describing them in a rights.csv
-file and using the :ref:`Import metadata <import-metadata>` feature.
+Archivematica supports `PREMIS`_ metadata and currently implements `version 3`_.
+It records actions (known as Events) taken on Objects by Agents. It also
+supports the addition of PREMIS Rights metadata. This page discusses how PREMIS
+is implemented in the AIP METS file.
+
+Skip to:
+
+* :ref:`PREMIS Events metadata <premis-events>`
+* :ref:`PREMIS Events list <event-list>`
+* :ref:`PREMIS Agent metadata <premis-agent>`
+* :ref:`PREMIS Rights metadata <premis-rights>`
+
+.. _premis-events:
+
+PREMIS Events metadata
+======================
+
+PREMIS Events will be recorded in separate digiprovMD sectons in the METS (see
+:ref:`METS in Archivematica <METS_schema>` for more information). PREMIS Events
+recorded in the METS will contain metadata for *most* of the following semantic
+units and semantic components (only if applicable):
+
+* eventIdentifier - eventIdentifierType
+* eventIdentifier - eventIdentifierValue
+* eventType
+* eventDateTime
+* eventDetail
+* eventOutcomeInformation - eventOutcome
+* eventOutcomeDetail - eventOutcomeDetailNote
+* linkingAgentIdentifier - linkingAgentIdentifierType
+* linkingAgentIdentifier - linkingAgentIdentifierValue
+
+For more information on semantic units and semantic components in PREMIS, please
+see the `Data Dictionary`_. Below is an example of a PREMIS Event for file
+format identification.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 2551-2581
```

Comment:
Yes, love seeing the use of scripts done this way!

Reply:
me too! I found it so much easier in the end because I could keep reusing the same few files!